client-app/src/pages/account/store/modules/orders/state.ts

```diff
@@ -0,0 +1,22 @@
+import { CustomerOrder } from '@common/api/api-clients';
+import { Pagination } from '@common/models/pagination';
+
+const initialColumns = [
+  {key: "number", label:"Order ID"},
+  {key: "status", label:"Status"},
+  {key: "createdDate", label:"Date"},
+  {key: "items.length", label:"# Items"},
+  {key: "createdBy", label:"Created by"},
+  {key: "total.formattedAmount", label:"Total"},
+];
+
+export class OrdersState{
+  constructor( public ordersPage: CustomerOrder[] = [],
```

Comment:
Why are you calling it ordersPage, if it just orders (and you use this name below)?

Reply:
Ok. I`ll name this as "orders".